avocado/utils/cpu.py

```diff
@@ -188,3 +189,56 @@ def offline(cpu):
     if _get_cpu_status(cpu):
         return 1
     return 0
+
+
+def get_cpu_idle():
```

Comment:
The name is quite hard to grasp. How about `get_cpu_idle_states` or even `get_cpuidle_states` (as in sys the name is joined into cpuidle. It might help associating the name to feature.